nntrainer/layers/multi_head_attention_layer.cpp

```diff
@@ -495,15 +498,10 @@ void MultiHeadAttentionLayer::forwarding(RunLayerContext &context,
 
   causal_mask.setZero();
 
-#ifdef ENABLE_FP16
-#define _MASK_NUM -1e4
-#else
-#define _MASK_NUM -1e10
-#endif
-
   for (unsigned int i = 0; i < mask_dim_height; ++i) {
     for (unsigned int j = i + 1; j < mask_dim_width; ++j) {
-      causal_mask.setValue(0, 0, i, j, _MASK_NUM);
+      causal_mask.setValue(
+        0, 0, i, j, _MASK_NUM(attention_weight.getTensorType().data_type));
```

Comment:
how about using `getDataType()` for simplicity
```suggestion
        0, 0, i, j, _MASK_NUM(attention_weight.getDataType()));
```